js/index.js

```diff
@@ -25,4 +25,120 @@ menuIcon.addEventListener('click', openMenuModal);
+  {
+    id: 2,
+    name: 'Uber Navigation',
+    description: 'A smart assistant to make driving more safe, efficient, and fun by unlocking your most expensive computer: your car.',
+    image: { src: '../assets/snapshoot-2.png', title: 'Portfolio' },
+    meta: ['Uber', 'Lead Developer', '2018'],
+    technologies: ['html', 'css', 'Python', 'javascript'],
+    sourceLink: 'https://github.com/BB-Simon',
+    liveLink: 'https://bb-simon.github.io/',
+  },
+  {
+    id: 3,
+    name: 'Tonic',
+    description: 'A daily selection of privately personalized reads; no accounts or sign-ups required.',
+    image: { src: '../assets/snapshoot-3.png', title: 'Portfolio' },
+    meta: ['Uber', 'Back End Dev', '2018'],
+    technologies: ['html', 'css', 'Nodejs', 'javascript'],
+    sourceLink: 'https://github.com/BB-Simon',
+    liveLink: 'https://bb-simon.github.io/',
+  },
+  {
+    id: 4,
+    name: 'Multi-Post Stories',
+    description: 'A smart assistant to make driving more safe, efficient, and fun by unlocking your most expensive computer: your car.',
+    image: { src: '../assets/snapshoot-4.png', title: 'Portfolio' },
+    meta: ['CANOPY', 'Full Stack Dev', '2015'],
+    technologies: ['html', 'css', 'Ruby on rails', 'javascript'],
+    sourceLink: 'https://github.com/BB-Simon',
+    liveLink: 'https://bb-simon.github.io/',
+  },
+];
+
+// Listener's functions
+const createProjectCard = (project) => {
+  const cardContainer = document.createElement('div');
+  if (project.id === 1 || project.id === 3) {
+    cardContainer.classList.add('card-conntainer', 'flex-reverse');
+  } else {
+    cardContainer.classList.add('card-conntainer');
+  }
+
+  const techList = project.technologies.map((tech) => `<li class="card-category">${tech}</li>`);
+  const metaList = project.meta.map((meta, index) => {
+    if (index !== 0) {
+      return `<div class="divider"></div>
+        <span class="card-meta-item muted">${meta}</span>`;
+    }
+    return `<span class="card-meta-item">${meta}</span>`;
+  });
+
+  const html = `
+    <img src=${project.image.src} alt=${project.image.title}>
+    <div class="card-content">
+        <h2 class="card-title">${project.name}</h2>
+        <div class="card-meta">
+          ${metaList.join('')}
+        </div>
+        <p id="card-description">${project.description}</p>
+        <ul class="card-categories-wrapper">
+          ${techList.join('')}
+        </ul>
+        <div class="card-cta-wrapper">
+            <button class="card-btn see-project">See Project</button>
+        </div>
+    </div>`;
+  cardContainer.innerHTML = html;
+  worksSection.appendChild(cardContainer);
+};
+
+projectData.forEach((project) => createProjectCard(project));
+const cardBtn = document.querySelectorAll('.see-project');
+const modalOverlay = document.getElementById('project-detail-modal-overlay');
+const projectModal = document.getElementById('project-detail-modal');
+const projectCloseBtn = document.getElementById('project-close-btn');
```

Comment:
- Great job on your design 👍👍 I think you have to work to make a see a project button popup with the exact detail of the card as your previous reviewer suggested to you.
- I think your main problem is not looking through the`see project` buttons
```
 seeProject.forEach((btn, i) => {
    seeProject[i].addEventListener('click', () => {
      openModal(i);
      });
    });
```
you ca do like that 👌👌👌